src/src/sentry.filter.ts

```diff
@@ -0,0 +1,12 @@
+import { Catch, ArgumentsHost } from '@nestjs/common';
```

Comment:
1. how this is different from
~/src/src/common/interceptors/sentry.interceptor.ts?

2. do we need both?

3. if we only need this move to ~/src/src/common/filters

Reply:
it is working without sentry global filter so I removed sentry global filter. Also as we discussed, I also removed transactions from 3 cron job.